### ISSUE TITLE
Is the name okay?

### ISSUE THREAD

**Issue by SimonBiggs** [libreapp/libreapp#1]
Chose something that isn't locked into python, isn't locked into medical physics. Also, amazingly the name has no big hits on Google.

It also results in two quite neat website names:

LibreApp.net and LibreApp.store

Libre is what is often used to make it clear that software is free in terms of free to do with it what ever you want, as opposed to no cost. Libre not gratis.

**Comment by SimonBiggs** [comment]
Picked up those website names, just so they don't get snatched from under our noses by some web crawler bot that sits on in purchased sites.

**Comment by darcymason** [comment]
There is, however, this: http://wiki.p2pfoundation.net/LibreApp for open scientific publication, but the referenced link http://www.libreapp.org/ gives timeout.

I like the name well enough, but did wonder whether it needs something to do with radiotherapy or physics/imaging etc. to make it more SEO-friendly.  But then I realize the major vendors do not have those kinds of words in their names.  In fact I'm growing to like it more as it can kind of stand out in the branding sense, as you said, free to do with it what you want.

**Comment by SimonBiggs** [comment]
Yeah, and the more I think about it... Medical Physics isn't the only field that would benefit from an application like this

We certainly should not spread ourselves thin. Focus on Medical Physics alone, but not lock ourselves out of branching out to another niche should it seem like a good idea.

**Comment by darcymason** [comment]
> Yeah, and the more I think about it... Medical Physics isn't the only field that would benefit from an application like this

I was thinking so too - so perhaps there is a libreapp.store/medphys or libreapp.store/health/rt or something like that, at least eventually.   Yeah, but not too thin.

**Comment by SimonBiggs** [comment]
Yup, makes sense. And the libreapp.store and libreapp.net sites will use
the same logins etc so when something is selected within the store it then
appears within the .net app.
